docs/endpoints/openrtb2/auction.md

```diff
@@ -226,6 +226,28 @@ A typical `storedrequest` value looks like this:
 
 For more information, see the docs for [Stored Requests](../../developers/stored-requests.md).
 
+#### Cache bids
+
+Bids can be temporarily cached on the server by sending the following data as `request.ext.prebid.cache`:
+
+```
+{
+  "bids": {}
+}
+```
+
+If `request.ext.prebid.targeting` was also in the request, then Prebid Server will make a _best effort_ to
+include these extra `bid.ext.prebid.targeting` keys:
+
```

Comment:
Should probably put in something to indicate how to find the cache keys if targeting parameters are not set, or explicitly call out that caching is not supported in the case that targeting is not requested.

Reply:
That sounds good. For now, caching is not supported when targeting is not requested. I definitely expect someone to add this feature in the future though.